airflow/executors/local_executor.py

```diff
@@ -43,40 +42,44 @@
 This option could lead to the unification of the executor implementations, running
 locally, into just one `LocalExecutor` with multiple modes.
 """
-
-import multiprocessing
 import subprocess
-from queue import Empty
-
-from airflow.executors.base_executor import BaseExecutor
+from multiprocessing import Manager, Process
+from multiprocessing.managers import SyncManager
+from queue import Empty, Queue  # pylint: disable=unused-import  # noqa: F401
+from typing import Any, List, Optional, Tuple, Union  # pylint: disable=unused-import # noqa: F401
+
+from airflow import AirflowException
+from airflow.executors.base_executor import NOT_STARTED_MESSAGE, PARALLELISM, BaseExecutor, CommandType
+from airflow.models.taskinstance import (  # pylint: disable=unused-import # noqa: F401
+    TaskInstanceKeyType, TaskInstanceStateType,
+)
 from airflow.utils.log.logging_mixin import LoggingMixin
 from airflow.utils.state import State
 
+# This is a work to be executed by a worker.
+# It can Key and Command - but it can also be None, None which is actually a
+# "Poison Pill" - worker seeing Poison Pill should take the pill and ... die instantly.
+ExecutorWorkTodo = Tuple[Optional[TaskInstanceKeyType], Optional[CommandType]]
```

Comment:
```suggestion
ExecutorWorkType = Tuple[Optional[TaskInstanceKeyType], Optional[CommandType]]
```

I think?

Reply:
Interesting TYP(E)O. Too much to do recently maybe :).